HealthcareBase/Repository/UsersRepository/EmployeesAndPatientsRepository/Interface/MedicalRecordRepository.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Model.Users.Patient.MedicalHistory;
+using Repository.Generics;
+
+namespace Repository.UsersRepository.EmployeesAndPatientsRepository
+{
+    public interface MedicalRecordRepository
+    {
+        
```

Comment:
Rename all interfaces to start with "I" like (IMedicalRecordRepo). This repeats in few repos.

Reply:
All interfaces renamed.